aliases/available/vagrant.aliases.bash

```diff
@@ -2,13 +2,16 @@ cite 'about-alias'
 about-alias 'vagrant aliases'
 
 # Aliases
+alias vhl='vagrant hosts list'
+alias vscp='vaagrant scp'
+alias vsl='vagrant snapshot list'
+alias vst='vagrant snapshot take'
 alias vup="vagrant up"
 alias vupl="vagrant up 2>&1 | tee vagrant.log"
 alias vh="vagrant halt"
 alias vs="vagrant suspend"
 alias vr="vagrant resume"
 alias vrl="vagrant reload"
-alias vssh="vagrant ssh"
```

Comment:
Any reason for deleting this one? I use `vssh` quite a bit, and don't want to see it go...